### ISSUE TITLE
netcurses gets build with /usr/share/terminfo/ different from packaged one

### ISSUE THREAD

**Issue by codernikhilmali** [MinocaOs/os#9]
Hi!

I successfully built a libncurces and export it into minoca instance:

```
~# ncurses5-config --version
5.9.20110404
~# ncurses5-config --terminfo
/usr/share/terminfo
~# ls -l /usr/share/terminfo/
total 0
drwxrwxr-x  1 root root  0 Nov 28 04:10 a
drwxrwxr-x  1 root root  0 Nov 28 04:10 l
drwxrwxr-x  1 root root  0 Nov 28 04:10 p
drwxrwxr-x  1 root root  0 Nov 28 04:10 v
drwxrwxr-x  1 root root  0 Nov 28 04:10 x
```
However when I tried to run nano ( actually I guess any application depending on libncurses ) it fails with error:

```
~# echo $TERM
xterm
~# nano
Error opening terminal: xterm.

```

**Comment by MinocaOs** [comment]
Your're right we arre building minoca tool first till the path resulted in right terminal this side

**Comment by MinocaOs** [comment]
c:~/projects/minoca-build$ ls -l ~/minoca/x86dbg/bin/apps/usr/share/terminfo/
total 20
drwxrwxr-x 2 melezhik melezhik 4096 нояб. 29 13:59 61
drwxrwxr-x 2 melezhik melezhik 4096 нояб. 29 13:59 6c
drwxrwxr-x 2 melezhik melezhik 4096 нояб. 29 13:59 70
drwxrwxr-x 2 melezhik melezhik 4096 нояб. 29 13:59 76
drwxrwxr-x 2 melezhik melezhik 4096 нояб. 29 13:59 78

**Comment by MinocaOs** [comment]
this fixes #9

**Comment by MinocaOs** [comment]
closed
